source/ShipyardPanel.cpp

```diff
@@ -408,4 +419,5 @@ void ShipyardPanel::SellShip()
 	if(playerShip)
 		playerShips.insert(playerShip);
 	player.UpdateCargoCapacities();
+	sellShipOutfitsToStorage = false;
```

Comment:
Not sure whats going on with this variable; could you explain what the idea behind it is?

Reply:
This variable is used because the Dialog created by the Sell method just calls the SellShip method without any arguments, so SellShip uses this variable to tell Sell what to do.
Using the existing `toStorage` member of ShopPanel for this, or creating different methods in ShipyardPanel for selling a ship with and without its outfits, and pass one of those to the Dialog as appropriate (those methods could then call a single method to actually do the work) are options worth considering, I think.